app/src/main/java/com/infomaniak/swisstransfer/ui/screen/main/transferdetails/TransferDetailsViewModel.kt

```diff
@@ -67,16 +75,24 @@ class TransferDetailsViewModel @Inject constructor(
 ) : ViewModel() {
 
     private val _transferUuidFlow = MutableSharedFlow<String>(1)
-
     @OptIn(ExperimentalCoroutinesApi::class)
-    val uiState = _transferUuidFlow
+    private val transferUuidFlow : Flow<TransferDetailsUiState> = _transferUuidFlow
```

Comment:
Your code is not properly formatted

```suggestion
    private val transferUuidFlow: Flow<TransferDetailsUiState> = _transferUuidFlow
```